user-guide/Advanced_Modules/User Defined APIs/UD_APIs_UserDefinableApiEndpoint.md

```diff
@@ -84,7 +85,8 @@ For example, this is the default configuration:
 }
 ```
 
-IIS also has a rewrite rule (Reroute User Definable APIs) that forwards API requests to the port used by *UserDefinableApiEndpoint* (5002). **When you specify a custom port in *appsettings.custom.json*, you will also have to update this rule**:
+> [!IMPORTANT]
+> IIS also has a rewrite rule (Reroute User Definable APIs) that forwards API requests to the port used by *UserDefinableApiEndpoint* (5002). **When you specify a custom port in *appsettings.custom.json*, you will also have to update this rule**:
```

Comment:
Checking this in docs it looks like something's missing. [Screenshot](https://skylinebe-my.sharepoint.com/:i:/g/personal/peter_vanpoucke_skyline_be/Ef3kQ-iOZBNEvALpzsOl4d4BRoLqsOL0y7cHH9IgPQWqFA?e=5415zD)

S:
> > **When you specify a custom port in *appsettings.custom.json*, you will also have to update the rule there**.

> To update the IIS rewrite rule:

Reply:
```suggestion
> IIS also has a rewrite rule (Reroute User Definable APIs) that forwards API requests to the port used by *UserDefinableApiEndpoint* (5002). **When you specify a custom port in *appsettings.custom.json*, you will also have to update this rule.**
```
You can update the rewrite rule by following these steps: